fides/values.yaml

```diff
@@ -72,6 +72,8 @@ privacyCenter:
     configJsonPath: config/privacyCenterConfig.json
     # privacyCenter.configuration.configCSSPath specifies the location of the config.css file to override the default styles.
     configCSSPath: config/privacyCenterConfig.css
+    # privacyCenter.configuration.configFilesOverride specifies the name of an existing configmap with the keys config.css and config.json containing the customization files.
```

Comment:
nit: I don't _quite_ understand how this would work, reading it here. If set, I think it will supercede the values for `configJsonPath` and `configCSSPath` above, right?

Reply:
Correct, It just won't use the other paths. This is needed because if you use this as a subchart, the Files.get method can't access anything outside of the subchart itself.